app/controllers/orders_controller.rb

```diff
@@ -1,5 +1,5 @@
 class OrdersController < ApplicationController
-  include CurrentCart
+  include CurrentCart, CurrentUser
```

Comment:
Just for readability and avoid confusion\

```suggestion
  include CurrentUser
  include CurrentCart
```

Reply:
done